CHANGELOG.md

```diff
@@ -146,17 +125,26 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- **CUMULUS-3070**
+  - Fixed inaccurate typings for PostgresGranule in @cumulus/db/types/granule
+  - Fixed inaccurate typings for @cumulus/api/granules.ApiGranule and updated to
+    allow null
 - **CUMULUS-3104**
   - Fixed TS compilation error on aws-client package caused by @aws-sdk/client-s3 3.202.0 upgrade
+- **CUMULUS-3116**
+  - Reverted the default ElasticSearch sorting behavior to the pre-13.3.0 configuration
+  - Results from ElasticSearch are sorted by default by the `timestamp` field. This means that the order
+  is not guaranteed if two or more records have identical timestamps as there is no secondary sort/tie-breaker.
 
 ## [v13.4.0] 2022-10-31
 
-
 ### Notable changes
 
 - **CUMULUS-3104**
   - Published new tag [`43` of `cumuluss/async-operation` to Docker Hub](https://hub.docker.com/layers/cumuluss/async-operation/43/images/sha256-5f989c7d45db3dde87c88c553182d1e4e250a1e09af691a84ff6aa683088b948?context=explore) which was built with node:14.19.3-buster.
 
+### Added
```

Comment:
So this is a change to the 13.4.0 release notes. It looks right. The notable change seems like it was 3104 and the rest of this should just be "added". Just want to verify that this was the intention.

Reply:
This section has been there with a few updates in this additional PR #[3188](https://github.com/nasa/cumulus/pull/3188) from @jennyhliu  as to help sorting out the correct order of things in this release CHANGELOG. Hence, I would assume it's meant to be there.